data-prepper-plugins/peer-forwarder/src/test/java/com/amazon/dataprepper/plugins/processor/peerforwarder/PeerClientPoolTest.java

```diff
@@ -16,4 +30,36 @@ public void testGetClientValidAddress() {
 
         assertNotNull(client);
     }
+
+    @Test
+    public void testGetClientWithSSL() {
```

Comment:
Is there no test for client without ssl?

Reply:
testGetClientValidAddress is w/o ssl by default. But there is no connect to server test on that. Will add one for completeness.